front/app/components/AuthProviders/index.tsx

```diff
@@ -33,6 +33,7 @@ const Container = styled.div`
   display: flex;
   flex-direction: column;
   align-items: stretch;
+  width: 100%;
```

Comment:
Fixes 
<img width="757" alt="DONE - full-screen sign in width" src="https://user-images.githubusercontent.com/16427929/225950555-1e837640-f445-4a21-9dba-cda625de35b3.png">

Reply:
It's full-width now.